vng_api_common/views.py

```diff
@@ -183,6 +183,9 @@ def _test_nrc_config(check_autorisaties_subscription=True) -> list:
     nrc_config = NotificationsConfig.get_solo()
     nrc_client: Optional[Client] = NotificationsConfig.get_client()
 
+    if not nrc_client:
```

Comment:
Although it might be nice to return early here, looking at the code it currently returns [this code](https://github.com/maykinmedia/commonground-api-common/pull/47/files#diff-65e65d396ca83c1d0b6628e17ba2c9b13a8e6b4b0b6e2d59de51579951664f90R246) whenever no client was configured. Unless we want to have different behavior I think we should return that here too or remove this path which returns early.

Reply:
I added this because I saw the change here: https://github.com/open-zaak/open-zaak/commit/08c7ccd3ebc8c8ef146eb06290f58284723beb34

Though I'm not sure if we need it at all?

Reply:
discussed on slack: replaced it with the current behavior in OZ